Commands/notify.py

```diff
@@ -119,13 +145,9 @@ async def show_lists(self, ctx):
                 await msg.add_reaction(v["emoji"] if not v["is_custom_emoji"] else ctx.bot.get_emoji(int(v["emoji"])))
 
             # TODO make reaction time configurable
-            timeout = time.time() + 60*5  # 5 minutes from now
-            reaction_added_task = asyncio.create_task(
-                self.wait_for_added_reactions(ctx, msg, guild_data, timeout)
-            )
-            reaction_removed_task = asyncio.create_task(
-                self.wait_for_removed_reactions(ctx, msg, guild_data, timeout)
-            )
+            timeout = time.time() + 60 * 5  # 5 minutes from now
```

Comment:
Can we put this in constants.py?

Reply:
Yes, but I would rather not do it on this branch because constants.py is already created in another branch so it makes more sense to me to move this variable once that branch is also merged to develop.

Reply:
Sounds fair